### PR TITLE
feat: switch various components to use `@brightspace-ui/core`

### PR DESCRIPTION
**Pull request by devpow112**
All of these were simply drop in replacements as the files from the individual packages simply had a single line that was an import from `@brightspace-ui/core`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,12 +47,9 @@
   },
   "dependencies": {
     "@brightspace-ui/core": "^1",
-    "d2l-alert": "BrightspaceUI/alert#semver:^4",
-    "d2l-hypermedia-constants": "^6.30.0",
-    "d2l-inputs": "BrightspaceUI/inputs#semver:^2",
```
Comment (devpow112) [inline package.json:52]: This wasn't actually used anywhere.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -6,14 +6,18 @@ jobs:
...
       - name: Install dependencies
-        run: npm install
+        run: npm i
       - name: Lint
         run: npm run lint
       - name: Unit Tests (SauceLabs)
         run: npm run test:sauce
         env:
-          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY_FARNSWORTH }}
-          SAUCE_USERNAME: D2L_Farnsworth
```
Comment (dlockhart) [inline .github/workflows/ci.yml:19]: There are separate Sauce accounts that help spread the VMs around. Farnsworth isn't really a team anymore though so maybe this makes sense.

**Review by dlockhart — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (devpow112) [reply .github/workflows/ci.yml:19]: Yeah that was my thoughts here. I can revert this if you like.

Comment (dlockhart) [reply .github/workflows/ci.yml:19]: Nah it's OK as it is I think. I'll remove the secret from the repo settings.

**Pull request merged**